### ISSUE TITLE
LimeChat 2.35

### ISSUE THREAD

**Issue by chonthu** [ryanflorence/limescripts#7]
Doesnt work.

**Comment by ryanflorence** [comment]
fixed.

**Comment by ryanflorence** [comment]
I'm just kidding. What doesn't work? Is there an error? What happens?

**Comment by chonthu** [comment]
Looks like Limechat is not reading the js file

**Comment by ryanflorence** [comment]
I wonder if your file is named correctly? there are two sets of themes, some "system" themes and then user themes, sometimes they have the same name. I don't think it picks up custom JS for the system ones.

**Comment by chonthu** [comment]
![Screen Shot 2013-04-26 at 1 28 22 PM](https://f.cloud.github.com/assets/339132/431900/be8bb824-ae96-11e2-9932-a7942e0dd226.png)

Im using it with a custom theme, please see my screenshot, maybe i didn't put it in the right folder?

**Comment by chonthu** [comment]
Any Ideas?

**Comment by jenseng** [comment]
assuming `Themes` is inside `~/Library/Application Support/LimeChat` and those js symlinks are correct, it should work. 

you may want to confirm you're actually using your custom theme. for example, i use the Spring Night theme. note that I have the bottom one selected, rather than the built-in one above it: http://cl.ly/image/303E2A1U0V16/Screen%20Shot%202013-05-01%20at%2010.26.55%20AM.png

**Comment by chonthu** [comment]
Yes I've confirmed both with no luck. What version of Limechat are you using? LimeChat 2.35 ?
